config/default/metrics_service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
Do we need the service? Or is this just a nice to have?

Reply:
Nice to have, so people don't have to create it themselves.